### ISSUE TITLE
recall differecne moblienet-yolov3-lite between tiny-yolov3

### ISSUE THREAD

**Issue by lqian** [eric612/MobileNet-YOLO#133]
Hi Eric,

I train one class detection net on the same dataset. training mobilenet-yolov3-lite with the project and train tiny-yolov3 with pjreddie/darknet. The train log show mobilenet-yolov3-lite recall/catalog rate **0.898755** after 16000 iterations. However, darknet tiny-yolov3 achieves **> .84 IOU and > .99 recall** with same training iterations.

Hera are moblienet-yolov3-lite log.
`
I0711 17:29:02.647851 18056 solver.cpp:442] Iteration 16000, Testing net (#0)
I0711 17:29:02.647929 18056 net.cpp:678] Ignoring source layer label_data_1_split
I0711 17:29:02.648034 18056 net.cpp:678] Ignoring source layer Yolov3Loss1
I0711 17:29:02.648039 18056 net.cpp:678] Ignoring source layer Yolov3Loss2
I0711 17:29:12.833758 18056 blocking_queue.cpp:50] Waiting for data
I0711 17:29:27.903025 18056 blocking_queue.cpp:50] Waiting for data
I0711 17:29:42.145201 18056 blocking_queue.cpp:50] Waiting for data
I0711 17:29:57.373795 18056 blocking_queue.cpp:50] Waiting for data
I0711 17:30:02.651219 18056 solver.cpp:549] class1: 0.898755
I0711 17:30:02.651268 18056 solver.cpp:555]     Test net output #0: detection_eval = 0.898755
I0711 17:30:05.275408 18056 solver.cpp:252] Iteration 16000 (0.0992634 iter/s, 100.742s/10 iters), loss = 0.443669
I0711 17:30:05.275462 18056 solver.cpp:271]     Train net output #0: det_loss1 = 0.0391493 (* 1 = 0.0391493 loss)
I0711 17:30:05.275471 18056 solver.cpp:271]     Train net output #1: det_loss2 = 0.215538 (* 1 = 0.215538 loss)
I0711 17:30:05.275480 18056 sgd_solver.cpp:121] Iteration 16000, lr = 0.00025
I0711 17:30:08.207433 18056 yolov3_layer.cpp:411] noobj: 0.00154096 obj: 0.700904 iou: 0.792483 cat: 0.99998 recall: 1 recall75: 0.875 count: 1
I0711 17:30:08.224016 18056 yolov3_layer.cpp:411] noobj: 0.000941287 obj: 0.706208 iou: 0.699302 cat: 0.999995 recall: 0.875556 recall75: 0.494444 count: 4
I0711 17:30:15.871816 18056 yolov3_layer.cpp:411] noobj: 0.00115902 obj: 0.559301 iou: 0.799716 cat: 0.999983 recall: 1 recall75: 0.75 count: 2
I0711 17:30:15.885015 18056 yolov3_layer.cpp:411] noobj: 0.000701723 obj: 0.730736 iou: 0.74864 cat: 0.999996 recall: 0.886667 recall75: 0.613333 count: 4
I0711 17:30:22.797222 18056 yolov3_layer.cpp:411] noobj: 0.00155675 obj: 0.695304 iou: 0.789466 cat: 0.999996 recall: 1 recall75: 0.791667 count: 1
I0711 17:30:22.810726 18056 yolov3_layer.cpp:411] noobj: 0.000823395 obj: 0.753544 iou: 0.747108 cat: 0.999997 recall: 0.94 recall75: 0.55 count: 4
I0711 17:30:29.610175 18056 yolov3_layer.cpp:411] noobj: 0.00203038 obj: 0.711658 iou: 0.821948 cat: 0.999985 recall: 1 recall75: 0.846154 count: 1
I0711 17:30:29.626137 18056 yolov3_layer.cpp:411] noobj: 0.000920128 obj: 0.733286 iou: 0.702275 cat: 0.999997 recall: 0.87 recall75: 0.512222 count: 4
I0711 17:30:36.227602 18056 yolov3_layer.cpp:411] noobj: 0.00250009 obj: 0.754275 iou: 0.810595 cat: 0.999988 recall: 1 recall75: 0.705128 count: 2
I0711 17:30:36.236629 18056 yolov3_layer.cpp:411] noobj: 0.000708616 obj: 0.689389 iou: 0.703718 cat: 0.999996 recall: 0.784921 recall75: 0.539841 count: 4
I0711 17:30:43.026089 18056 yolov3_layer.cpp:411] noobj: 0.00177928 obj: 0.822632 iou: 0.758352 cat: 0.999987 recall: 0.923077 recall75: 0.653846 count: 1
I0711 17:30:43.036728 18056 yolov3_layer.cpp:411] noobj: 0.000726599 obj: 0.703824 iou: 0.714286 cat: 0.999997 recall: 0.821111 recall75: 0.584444 count: 4
I0711 17:30:44.558761 18056 solver.cpp:252] Iteration 16010 (0.254562 iter/s, 39.2832s/10 iters), loss = 0.498537
I0711 17:30:44.558830 18056 solver.cpp:271]     Train net output #0: det_loss1 = 0.0276054 (* 1 = 0.0276054 loss)
I0711 17:30:44.558840 18056 solver.cpp:271]     Train net output #1: det_loss2 = 0.55703 (* 1 = 0.55703 loss)
`

darknet training log
`
Region 16 Avg IOU: 0.830850, Class: 0.999822, Obj: 0.957939, No Obj: 0.000274, .5R: 1.000000, .75R: 0.857143,  count: 7
Region 23 Avg IOU: 0.837271, Class: 0.998396, Obj: 0.938535, No Obj: 0.000295, .5R: 1.000000, .75R: 0.857143,  count: 14
Region 30 Avg IOU: 0.806637, Class: 0.999439, Obj: 0.640513, No Obj: 0.000033, .5R: 1.000000, .75R: 0.777778,  count: 9
Region 16 Avg IOU: 0.862747, Class: 0.999021, Obj: 0.826834, No Obj: 0.000376, .5R: 1.000000, .75R: 1.000000,  count: 7
Region 23 Avg IOU: 0.876576, Class: 0.999769, Obj: 0.884862, No Obj: 0.000284, .5R: 1.000000, .75R: 1.000000,  count: 15
Region 30 Avg IOU: 0.835545, Class: 0.999627, Obj: 0.597858, No Obj: 0.000022, .5R: 1.000000, .75R: 0.750000,  count: 8
Region 16 Avg IOU: 0.793781, Class: 0.999273, Obj: 0.617494, No Obj: 0.000336, .5R: 1.000000, .75R: 0.750000,  count: 8
Region 23 Avg IOU: 0.845829, Class: 0.999290, Obj: 0.754881, No Obj: 0.000294, .5R: 1.000000, .75R: 0.882353,  count: 17
Region 30 Avg IOU: 0.844751, Class: 0.999901, Obj: 0.928351, No Obj: 0.000023, .5R: 1.000000, .75R: 1.000000,  count: 5
Region 16 Avg IOU: 0.820707, Class: 0.999860, Obj: 0.909772, No Obj: 0.000347, .5R: 1.000000, .75RSaving weights to pr-backup/pr-plate-yolov3-tiny-3l.backup
: 0.857143,  count: 7
Region 23 Avg IOU: 0.842686, Class: 0.999751, Obj: 0.834223, No Obj: 0.000283, .5R: 1.000000, .75R: 0.937500,  count: 16
Region 30 Avg IOU: 0.863811, Class: 0.999833, Obj: 0.671521, No Obj: 0.000037, .5R: 1.000000, .75R: 0.875000,  count: 8
16000: 0.125506, 0.102164 avg, 0.001000 rate, 1.650374 seconds, 2048000 images
`

**Comment by eric612** [comment]
Maybe it is  the color format problem ?
https://blog.csdn.net/xunan003/article/details/94740569
